recipes/python/__init__.py

```diff
@@ -7,7 +7,7 @@
 class PythonRecipe(Recipe):
     version = "2.7.1"
     url = "https://www.python.org/ftp/python/{version}/Python-{version}.tar.bz2"
-    depends = ["hostpython", "libffi", ]
+    depends = ["hostpython", "libffi", "openssl"]
```

Comment:
There are many use cases for python to be used without openssl and it would make sense to only do this if openssl is included by the user. I believe openssl support will be picked up by python and compiled by python setup if it's already in the toolchain included by the user.